source/slang/slang-dxc-support.cpp

```diff
@@ -190,7 +193,9 @@ namespace Slang
             args[argCount++] = L"-enable-16bit-types";
         }
 
-        const String sourcePath = calcSourcePathForEntryPoint(endToEndReq, entryPointIndex);
+        List<Int> entryPointIndices2;
+        entryPointIndices.add(entryPointIndex);
+        const String sourcePath = calcSourcePathForEntryPoints(endToEndReq, entryPointIndices2);
```

Comment:
It would be a non-issue if we introduce the singular `calcSourcePathForEntryPoint` back as a wrapper, but I will note that the need to declare `entryPointIndices2` here is redundant. You could just use the `entryPointIndices` you already constructed earlier in the function.

Alternatively, you could wrap up these first call to `emitEntryPointsSource` in its own scope with `{ ... }` in order to narrow the lifetime of the first `entryPointIndices` variable. Either way, needing to use a variable name like `entryPointIndices2` doesn't feel clean.